### ISSUE TITLE
OUTDATED Documentation - Please update --> 15-Service-apply-for-example.md

### ISSUE THREAD

**Issue by fireheadman** [Icinga/icingaweb2-module-director#1009]
Trying to follow this document to create a http service, see link to issue:
[https://monitoring-portal.org/index.php?thread/41284-new-build-trying-to-create-default-icinga-service-in-director-http-disk/#post252434](url)

Per 15-Service-apply-for-example.md, the screenshots do not properly align to how director works now.   It also seems the screenshots are a compilation of two efforts...?  When I attempt to replicate the outcome I get different results:

For the sake of comparison, I used tcp_ports/tcp_port as the example has to prove/show the differences

Documentation:  (notice it says "Add new Icinga Service")
![image](https://user-images.githubusercontent.com/19847195/28184382-6ecfb5f4-67d0-11e7-82ae-126f349e9f58.png)
VS Mine: (notice it says "Add new Icinga Service template")
![image](https://user-images.githubusercontent.com/19847195/28184505-e57dca9c-67d0-11e7-9ce7-6ca2f9a025ae.png)

Documentation:  These seem identical  (NOTE: there is not any mention of clicking 'Store' back on the 'Service' Tab... I know this is assumed, but others new to Icinga may overlook this since it isn't present in the documenation nor on the 'Fields' screen.
![image](https://user-images.githubusercontent.com/19847195/28184630-5eb0f8da-67d1-11e7-8f62-d0b0f9463fa3.png)
VS Mine:
![image](https://user-images.githubusercontent.com/19847195/28184640-68f6c392-67d1-11e7-95e0-6c1c7ed7d4a1.png)

Documentation VS Mine  (These are identical)
![image](https://user-images.githubusercontent.com/19847195/28184659-7ded7566-67d1-11e7-8c16-a1871016f4e5.png)

Documentation: NOTE: The example all the sudden is importing "TCP_service"...but the previous screenshots are reflecting "Tcp Service Template".... This is highly confusing.   Also the example needs to expand out the Assignment Rules, which is now referred to as "Assign Where"....  So without this info, I cannot even complete/add the service as I do not have all the information.
![image](https://user-images.githubusercontent.com/19847195/28184754-dc73e282-67d1-11e7-8d5b-87438f54260d.png)

VS Mine:
![image](https://user-images.githubusercontent.com/19847195/28184848-37e4f318-67d2-11e7-8941-9085a9debc7e.png)
IF... I were to plug something in the "Assign where" field to allow me to save this...
![image](https://user-images.githubusercontent.com/19847195/28184943-a8a94126-67d2-11e7-9e44-50df175a0bdf.png)

I would end up with a config like this:
![image](https://user-images.githubusercontent.com/19847195/28184955-b97fac06-67d2-11e7-836e-050b8378be55.png)
VS 
Documentation:  Which totally doesn't match.
![image](https://user-images.githubusercontent.com/19847195/28184969-c707c2f0-67d2-11e7-9794-ca5cb587d94f.png)

I understand a lot of the documentation is out of date and there are limited resources in getting it updated... but at the same time, it really does make implementing Icinga a major challenge when the source documentation is lacking.

I'm willing to provide whatever help I can to you guys to make this better.

**Comment by Thomas-Gelf** [comment]
Yeah, those screen-shots cause a lot of work. And I'm about to change those forms once again :laughing:. Not kidding, will probably happen tomorrow. And in case I like the new ones it might be the right time to renew parts of the documentation.

We are working hard to get the `next` branch ready to be merged into `master`. It contains LOTS of changes. Currently `next` is 235 commits ahead of `master`, with the whole Diff being 23-24k lines. All tables, all dashlets, many forms have been touched. We usually try to always stay close to master, merge early and keep the master stable. This would not have been possible with all that refactoring, as we had to break a lot of things - so `next` diverged quite a bit.

As of this writing we are very closed to get this task finished. There are still some misbehaving forms, links or redirects pointing to the wrong target and so, but nothing to worry about. I would really like to close `next` within this week... we'll see.

Once we have all those shiny new things in the master branch we should be more or less ready to release v1.4.0. This would be a very good moment to renew parts of the documentation. Any help with improving it would be highly appreciated!

**Comment by sysadmama** [comment]
I'm working with ``next`` now for a while, and (for whatever reason) I have some experience in case of _writing for Director_. You could give me a poke as soon as the forms are ready for screenshots, and I'll see what I could do for the docs. Deal?

**Comment by Thomas-Gelf** [comment]
Deal :-)

**Comment by Thomas-Gelf** [comment]
Hint -> `next` is history and has been merged into `master`. Additionally I tagged 1.3.2, hasn't officially been announced yet. There will still be some small changes, but please expect the current master to become 1.4.0 very soon.
